scripts/build/gn_bootstrap.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+CHIP_ROOT="$(dirname "$0")/../.."
+
+source "$CHIP_ROOT/activate.sh"
```

Comment:
how is this script used?

Reply:
looks like maybe you're keeping this step just in case we add something later?

Reply:
This is for CI, the purpose is to account the time spent in bootstrap (which runs first time activate.sh is sourced) in a separate step. Will add a comment to that effect.

Currently the bootstrap is the most expensive part of build. Looking into if we can fix it with caching. Persistent CI runners would likely fix it because they could preserve the cache between builds.